tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleWorkspaceProject.java

```diff
@@ -555,7 +591,7 @@ private void _loadGradleWorkspaceProperties() {
 
 			IPath gradleProperties = projectLocation.append("gradle.properties");
 
-			properties = PropertiesUtil.loadProperties(gradleProperties);
+			properties.putAll(PropertiesUtil.loadProperties(gradleProperties));
```

Comment:
Do we need to clear the previous properties?  What if a user changes their properties?

Reply:
we still need to read build type from .balde.properties. I changed it to pullAll just want to keep content .blade.properties.